src/components/Organisms/ToDoCard/index.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import AddTaskButton from "../../Atoms/AddTaskButton/index";
+import Task from "../../Molecule/Task/index";
+import COLOR from "../../../variables/color";
+
+const TodoCard = () => {
+  const [taskList, setTaskList] = useState([]);
+
+  const onAddTaskButtonClick = () => {
+    setTaskList((taskList) => [...taskList, { name: "", initializing: true }]);
+  };
+
+  const onTaskComplete = (taskIndex) => {
+    const taskComplete = taskList.filter((task, index) => {
+      return index !== taskIndex;
+    });
+    setTaskList(taskComplete);
+  };
+
+  const onTaskNameChange = (taskIndex, value) => {
+    if (value === "") {
+      const editedTask = taskList.filter((task, index) => {
+        return index !== taskIndex;
+      });
+      setTaskList(editedTask);

```

Comment:
```suggestion
      const editedTaskList = taskList.filter((task, index) => {
        return index !== taskIndex;
      });
      setTaskList(editedTaskList);
```
`Task` というと単体のタスクが中に入っていることを想像しますが、今回は `Task` の配列なので、 `taskList` と配列であるということが直感的にわかる命名にしてあげると親切ですね！